tests/integration/test_store.py

```diff
@@ -263,20 +263,6 @@ def store_event():
     store_count.acquire(timeout=2)
 
 
-def test_store_not_normalized(mini_sentry, relay):
```

Comment:
The test is stale as normalization runs for all relays, including the default attributes (like `key_id`).